### PR TITLE
Refactor room type settings UI

### LINKED ISSUE #1101: Not possible to change room settings when creating a room
**Describe the bug**
When creating a room it is not possible to change the room settings

**To Reproduce**
1. Go to 'Räume'
2. Click on 'Raum erstellen'
3. Go to 'Standard Raumeinstellungen'
4. Try to toggle any setting.

**Expected behavior**
The settings change when clicking on the toggle buttons. However, they don't.

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser Chromium

### PR DESCRIPTION
**Pull request by samuelwei**
# Fixes #1101

## Type (Highlight the corresponding type)
- Bugfix
- Feature
- Documentation
- **Refactoring (e.g. Style updates, Test implementation, etc.)**
- Other (please describe):

## Checklist
- [x] Code updated to current develop branch head
- [x] Passes CI checks
- [x] Is a part of an issue
- [ ] Tests added for the bugfix or newly implemented feature, describe below why if not
- [ ] Changelog is updated
- [ ] Documentation of code and features exists

## Changes

- Refactor UI to show room type settings, replace interactive looking elements with static elements

## Screenshots of new design

image

image

### CODE REVIEW

**Comment by SamuelWei** [comment]
@tibroc What do you think?

File: resources/js/components/RoomTypeDetails.vue [hunk 1]
```diff
@@ -20,308 +20,20 @@
...
+                :key="field.key"
+                :id="field.key"
+                :value="roomType[field.key+'_default']"
+                :enforced="roomType[field.key+'_enforced']"
+                :label="field.label"
+                :roomType="roomType"
+                :type="field.type"
+                :options="field.options"
+              />
             </div>
```
Comment (Sabr1n4W) [inline resources/js/components/RoomTypeDetails.vue:37]:
```suggestion
            <!-- Show all room settings grouped by category -->
            <div v-for="settingGroup in roomTypeSettings" :key="settingGroup.title">
              <h4 class="my-2">{{ settingGroup.title }}</h4>
              <RoomTypeSettingsField
                v-for="setting in settingGroup.settings"
                :key="setting.key"
                :id="setting.key"
                :value="roomType[setting.key+'_default']"
                :enforced="roomType[setting.key+'_enforced']"
                :label="setting.label"
                :roomType="roomType"
                :type="setting.type"
                :options="setting.options"
              />
            </div>
```

Maybe change setting to settingGroup or something similar and field to setting

**Review by Sabr1n4W — CHANGES_REQUESTED** [review]
Maybe also change "fas fa-times" to "fa-solid fa-xmark" in RoomTypeCompareSettingsField and in RoomTypeSettingsField

**Comment by tibroc** [comment]
Looks a lot better imo! :rocket:

**Pull request merged**